apps/web/pages/api/integrations.ts

```diff
@@ -30,7 +30,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   if (req.method == "DELETE") {
-    const id = req.body.id;
+    /** Get the id from the stripe_payment credential for this user */
+    const paymentCredential = await prisma.credential.findFirst({
```

Comment:
`const stripePaymentCredential` as above

Reply:
Also we can move this code below where paymentCredential its being used on line 44